src/class/cdc/cdc.h

```diff
@@ -216,15 +216,18 @@ typedef enum
 //--------------------------------------------------------------------+
 
 /// Header Functional Descriptor (Communication Interface)
+TU_PACK_STRUCT_BEGIN
```

Comment:
Yes, that is possible. The only exception, which comes to my mind at the moment, is that it could be a negative speed impact if accessing packed structures. Maybe we should check if the packed attribute is really required on all structs here ?

Reply:
we only group these packed structure that are consecutive to each other in tusb_types.h i.e we only group them together instead of declaring for each struct. So it is literally the same.